.github/workflows/ci.yml

```
@@ -61,3 +61,38 @@ jobs:
     - name: Documentation
       run: |
         cabal haddock
+
+  stack:
+      name: stack / ghc ${{ matrix.ghc }}
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          stack: ["2.7.5"]
+          ghc: ["9.2.3"]
+
+      steps:
+      - uses: actions/checkout@v2
```

The latest version of this action is `v3` so let's use it
```suggestion
      - uses: actions/checkout@v3
```